### ISSUE TITLE
Reduce "visual clutter" by using different style for inline code (without background color)?

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/sphinx_typo3_theme#176]
To me, some of our pages look "visually cluttered" (not sure what would be a better word). It is distracting, makes it less readable.

The reason, I think, are several, here in this issue I would like to focus on:

The inline code / text roles are often formatted with a darker background and rounded edges. In some cases this may be useful, but we use the inline formatting a lot and often this impedes readability. Personally, it hurts my eyes.

Please compare the 2 screenshots below from Symfony documentation and TYPO3.

Just using backticks, such as  is formatted like this: 

![image](https://user-images.githubusercontent.com/13206455/198592302-4d0b5f2b-6fc5-4cf6-a823-40773ce1a919.png)

etc. 

Also, there is sometimes excessive use of inline code formatting (textroles) and also the excessive use of notes. (I will only focus on the inline code here.) That is a question of how the documentation is written and the conventions, but sometimes other formatting methods might help such as using confval or others.

## Examples:

TYPO3

![image](https://user-images.githubusercontent.com/13206455/198593739-d6b7ee98-02bb-4d8c-9a18-647ec2b9b4df.png)

![image](https://user-images.githubusercontent.com/13206455/198597975-59567af8-a595-4dd7-9417-8a307073c3df.png)

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Database/RestrictionBuilder/Index.html

----

Symfony 

They also have less of the formatting, but I think in our case it is combination of both (too much and how it is styled).

Here, not darker background color is used.

![image](https://user-images.githubusercontent.com/13206455/198594458-0e7deedf-8f16-40c1-ba5e-d03f8a26463a.png)

https://www.doctrine-project.org/projects/doctrine-orm/en/2.13/reference/query-builder.html

**Comment by marble** [comment]
Yes, I agree, we should reduce visual noise there. Thanks for reporting and compiling the examples. I'll see what I can do in the next versions of the theme. I plan to first show a proposal here.

**Comment by marble** [comment]
I have removed the padding, which has the biggest effect. I did not remove the border but set it to white.
Looks much butter IMHO.

In normal text:
![grafik](https://github.com/TYPO3-Documentation/sphinx_typo3_theme/assets/307057/3489e595-60a0-44d5-b7d8-4eecfeddf5de)

In a striped table:
![grafik](https://github.com/TYPO3-Documentation/sphinx_typo3_theme/assets/307057/2c2125d4-9cd8-4795-8c92-995e016f8950)

In a definition:
![grafik](https://github.com/TYPO3-Documentation/sphinx_typo3_theme/assets/307057/3d962741-6c4c-413c-bab9-80f98c43c5fb)
